koku/masu/processor/expired_data_remover.py

```diff
@@ -154,6 +157,10 @@ def remove(self, simulate=False, provider_uuid=None, line_items_only=False):
                     )
             else:
                 removed_data = self._cleaner.purge_expired_report_data(simulate=simulate, provider_uuid=provider_uuid)
+                with ReportManifestDBAccessor() as manifest_accessor:
+                    # Remove expired CostUsageReportManifests
+                    expiration_date = self._calculate_expiration_date()
+                    manifest_accessor.delete_provider_cost_usage_reports_older_than(provider_uuid, expiration_date)
```

Comment:
we still need the `if not simulate:` logic here.

Reply:
added